lib/Perl/Critic/Policy/TooMuchCode/ProhibitLargeBlock.pm

```diff
@@ -46,7 +46,7 @@ This policy scan for large code blocks of the following type.
     do { ... };
 
 By default a large block is one with more than 10 statements. If
-you need an other limit, you can set the parameter
+you need another limit, you can set the parameter
```

Comment:
This is an interesting one. Based on [this discussion](https://english.stackexchange.com/questions/56435/an-other-vs-another) I think you could argue for either version here.